docs/application/native/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,343 @@
+
+![Body pose](./media/mediavision_pose_tizen_def.png)
+
+The Pose landmark detection models are available in Open Model Zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models) or public GitHub site such as [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile). The public pose models provide landmark information, such as the number of landmarks and locations. To use them correctly, you must map the information to landmarks based on the definition. For example, you can use the [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile), which provides 14 landmarks as follows:
+
+![Body pose](./media/mediavision_pose_public_model_def.png),
+
+In this model, `-1` denotes that there is no landmarks. Using this landmark information, you can create a mapping file. Suppose you create a mapping file with the name `pose_mapping.txt,` then you can populate the `pose_mapping.txt` file as follows:
+
+```
+1
+2
+-1
+3
+4
+5
+6
+7
+8
+-1
+9
+10
+11
+12
+13
+14
+```
+
+`1` denotes that the first landmark of the model corresponds to the first definition,  `MV_INFERENCE_HUMAN_POSE_HEAD`. `-1` at the 3rd denotes that there is no corresponding landmark `MV_INFERENCE_HUMAN_POSE_THORAX`. `3` at the 4th denotes that the 3rd landmark of the model corresponds to the 4th, `MV_INFERENCE_HUMAN_POSE_RIGHT_SHOULDER`. The following table shows how the public model works.
+
+**Table: Example of how  [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile) maps to the definition**
+
+| Value | Defintion | pose_mapping.txt |
+| - | - | - |
+| 1 | MV_INFERENCE_HUMAN_POSE_HEAD | 1 |
+| 2 | MV_INFERENCE_HUMAN_POSE_NECK | 2 |
+| 3 | MV_INFERENCE_HUMAN_POSE_THORAX | -1 |
+| 4 | MV_INFERENCE_HUMAN_POSE_RIGHT_SHOULDER | 3 |
+| 5 | MV_INFERENCE_HUMAN_POSE_RIGHT_ELBOW | 4 |
+| 6 | MV_INFERENCE_HUMAN_POSE_RIGHT_WRIST | 5 |
+| 7 | MV_INFERENCE_HUMAN_POSE_LEFT_SHOULDER | 6 |
+| 8 | MV_INFERENCE_HUMAN_POSE_LEFT_ELBOW | 7 |
+| 9 | MV_INFERENCE_HUMAN_POSE_LEFT_WRIST | 8 |
+| 10 | MV_INFERENCE_HUMAN_POSE_PELVIS | -1 |
+| 11 | MV_INFERENCE_HUMAN_POSE_RIGHT_HIP | 9 |
+| 12 | MV_INFERENCE_HUMAN_POSE_RIGHT_KNEE | 10 |
+| 13 | MV_INFERENCE_HUMAN_POSE_RIGHT_ANKLE | 11 |
+| 14 | MV_INFERENCE_HUMAN_POSE_LEFT_HIP | 12 |
+| 15 | MV_INFERENCE_HUMAN_POSE_LEFT_KNEE | 13 |
+| 16 | MV_INFERENCE_HUMAN_POSE_LEFT_ANKLE | 14 |
+
+
+Motion capture file (MoCap) includes the movements of objects or a person. There are various MoCap formats, but a well-known BVH (BioVision Hierarchy) file is supported in Media Vision. BVH file has Hierarchy structure to provide landmark information with landmarks' name, and the structure can be changed. It means that landmarks information is different from the definition. To use them correctly, you have to map the information to the landmarks based on the definition. For example, the [BVH file](/media/mediavision_pose_bvh_sample.bvh) describes a squat pose as follows:
```

Comment:
@nawab-samsung 
Question to "BVH (BioVision Hierarchy) ".
I thought SRI-B's style to write an abbreviation was "BioVision Hierarchy (BVH)".
Would you please check?

Reply:
@mijin85cho thanks,I checked many [sources](https://www.google.com/search?client=firefox-b-d&q=BVHfile), and the BVH file is called mostly and commonly by its abbrevation so kept it otherwise. But as you say, I will go with your suggestion. 

```suggestion
Motion capture file (MoCap) includes the movements of objects or a person. There are various MoCap formats, but a well-known BioVision Hierarchy (BVH) file is supported in Media Vision. BVH file has Hierarchy structure to provide landmark information with landmarks' name, and the structure can be changed. It means that landmarks information is different from the definition. To use them correctly, you have to map the information to the landmarks based on the definition. For example, the [BVH file](/media/mediavision_pose_bvh_sample.bvh) describes a squat pose as follows: